### PR TITLE
adding some new domains

### PR DESCRIPTION
**Pull request by kzieglerjmu**
number for regions in domains/aux-domains.csv still missing, I don't know how they are set

### CODE REVIEW

**Comment by larsbuntemeyer** [comment]
Thanks @kzieglerjmu , i'll have a look asap.

File: domains/aux-domains.csv [hunk 1]
```diff
@@ -5,4 +5,9 @@ region,short_name,long_name,nlon,nlat,ll_lon,ll_lat,dlon,dlat,pollon,pollat
...
 4,EUN-0275,EUCP Northern Europe,649,811,-9.99125,-0.20625,0.0275,0.0275,-162.0,39.25
 4,EUC-0275,EUCP Central Europe,361,401,-9.52375,-5.29375,0.0275,0.0275,-162.0,39.25
-4,SGAR-0275,Greater Alpine Region,289,271,-9.38625,-9.66625,0.0275,0.0275,-162.0,39.25
\ No newline at end of file
+4,SGAR-0275,Greater Alpine Region,289,271,-9.38625,-9.66625,0.0275,0.0275,-162.0,39.25
+4,GER-11,German Region,121,121,-10.89,-6.89,0.11,0.11,-162.0,39.25
+,WAF-11,Westafrican Region,361,271,-20.68,2.64,0.11,0.11,180.0,90.0
+,MSEA-11,Mainland South East Asia,145,181,92.34,7.12,0.11,0.11,180.0,80.0
+,MED-11,Mediterranean Region,487,321,-27.925,-25.925,0.11,0.11,198.0,39.25
+,CAS-11,Central Asian Region,257,271,-20.025,-20.605,0.11,0.11,-103.39,43.48
```
Comment (larsbuntemeyer) [inline domains/aux-domains.csv:13]: I see here a conflict to the original CORDEX `CAS-11` (https://github.com/remo-rcm/tables/blob/main/domains/cordex-domains.csv#L11). Is that a small subset, e.g., focusing some country?

Comment (larsbuntemeyer) [reply domains/aux-domains.csv:13]: grafik

Comment (kzieglerjmu) [reply domains/aux-domains.csv:13]: @larsbuntemeyer, yes it is a subset of the original CORDEX CAS-11, we adjusted for our irrigation analysis, maybe we can call it CASi-11, if calling it CAS-11 is a problem

Comment (kzieglerjmu) [reply domains/aux-domains.csv:13]: good idea

Comment (larsbuntemeyer) [reply domains/aux-domains.csv:13]: This is now handled in #4

**Comment by larsbuntemeyer** [comment]
superseeded by https://github.com/remo-rcm/tables/pull/4

**Pull request closed without merging**